Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
     "mirrord-layer",
     "mirrord-cli",
     "mirrord-macro",
+    "mirrord-common",
```

Comment:
Because we compile with workspaces need to add the directory to `./mirrord-agent/Dockerfile`